books/RayTracingTheRestOfYourLife.html

```diff
@@ -197,17 +226,73 @@
 On my computer, I get:
 
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-    Regular    Estimate of Pi = 3.14151480
-    Stratified Estimate of Pi = 3.14158948
+    Regular    Estimate of Pi = 3.141184000000
+    Stratified Estimate of Pi = 3.141460000000
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 </div>
 
 Interestingly, the stratified method is not only better, it converges with a better asymptotic rate!
 Unfortunately, this advantage decreases with the dimension of the problem (so for example, with the
-3D sphere volume version the gap would be less). This is called the *Curse of Dimensionality*. We
-are going to be very high dimensional (each reflection adds two dimensions), so I won't stratify in
-this book, but if you are ever doing single-reflection or shadowing or some strictly 2D problem, you
-definitely want to stratify.
+3D sphere volume version the gap would be less). This is called the *Curse of Dimensionality*. Ray
+tracing is a very high-dimenstional algorithm, where each reflection adds two new dimensions:
+$\phi_o$ and $\theta_o$. We won't be stratifying the output reflection angle in this book, simply
+because it is a little bit too complicated to cover, but there is a lot of interesting research
+currently happening in this space. But, as an intermediary, we'll be stratifying the locations of
+the sampling positions inside of each pixel.
+
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++
+            world = cornell_box();
```

Comment:
I've been representing omitted contextual filler with `...`. But don't worry, this will be hugely rewritten with the scene work, so I'll build from your changes.